src/main/java/org/opentripplanner/routing/graph/GraphRepository.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.routing.graph;
+
+import org.opentripplanner.datastore.DataSource;
+import org.opentripplanner.standalone.config.BuildConfig;
+import org.opentripplanner.standalone.config.RouterConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GraphRepository {
```

Comment:
We should find a more descriptive name and add Javadoc

Reply:
We discussed this - repository does seem to be the right abstraction but some readers might not make the connection. So Javadoc will give a very short statement of what it does, and maybe a link to an explanation. As a repository it should also contain the logic for loading (not just saving) Graphs from DataSources.

Reply:
About the GraphRepository and the SerializedGraphObject: I think the best solution is to merge the Repository into the SerializedGraph. I think that is better, since we do not have the concept of a Repository elsewhere in OTP. We have DataSources as a “thing”, and introducing Repositories will slightly overlap.